src/coreclr/jit/codegenxarch.cpp

```diff
@@ -10709,7 +10709,6 @@ void CodeGen::genZeroInitFrameUsingBlockInit(int untrLclHi, int untrLclLo, regNu
     assert(compiler->compGeneratingProlog);
     assert(genUseBlockInit);
     assert(untrLclHi > untrLclLo);
-    assert(compiler->getSIMDSupportLevel() >= SIMD_SSE2_Supported);
```

Comment:
Replacing this assert is not needed? Is the idea that if AVX2 is not supported, then the default is SSE2 now?

Reply:
The default has been SSE2 for many years (always for x64 and since around .NET Core 2.1 for x86). We require it and consider it part of the baseline ISA, so there is no need to check for its existence outside of supporting the `DOTNET_EnableSSE2=0` switch for HWIntrinsic importation.